vmdb/product/toolbars/ems_infra_center_tb.yaml

```diff
@@ -22,6 +22,11 @@
       :text: 'Edit this #{ui_lookup(:table=>"ems_infra")}'
       :title: 'Edit this #{ui_lookup(:table=>"ems_infra")}'
       :url: '/edit'
+    - :button: ems_infra_scale
```

Comment:
@Ladas : correct, we need a toolbar class and a button class, generic and specific and get the specific behaviors into those subclasses. 
On top of that some plugability -- an interface to define a button into a named toolbar or given context (Infra --> Host or Cloud --> Zone or whatever) to start with.